extensions/notion/src/components/forms/CreatePageForm.tsx

```diff
@@ -91,14 +117,25 @@ export function CreatePageForm({ mutate, launchContext, defaults }: CreatePageFo
                 }
               : undefined,
           });
+        } else {
+          await showHUD("Page created ✅", { popToRootType: PopToRootType.Immediate, clearRootSearch: true });
+        }
 
-          if (mutate) {
-            mutate();
-            useNavigation().pop();
-          }
+        if (mutate) {
+          await mutate();
+          useNavigation().pop();
+        } else {
+          reset(initialValues);
+          const titleProperty = databaseProperties?.find((dp) => dp.type == "title");
+          titleProperty && focus(createPropertyId(titleProperty));
```

Comment:
What is this code doing?

Reply:
Reset form. 
- If you open just CreatePageForm then I will clear all fields
- if you open CreatePageForm with quick link, then we have some default values, so will use these initial values.

Then I will focus on title field, because title of the page is required field in Notion. This make easier flow -> add one page -> save -> browser will focus first field and you are working on thext page